src/test/resources/pages/instructorSearchPageSearchCommentsAsHelper.html

```diff
@@ -1,151 +1,20 @@
-                  </td>
-                </tr>
-                <tr>
-                  <td>
-                    <strong>
-                      <span class="highlight">
-                        Response
-                      </span>
-                      :
-                    </strong>
-                    <span class="highlight">
-                      Response
-                    </span>
-                    from student 3 "to" student 2. Multiline test.
-                  </td>
-                </tr>
-                <tr class="active">
-                  <td>
-                    <span class="highlight">
-                      Comment
-                    </span>
-                    (s):
-                  </td>
-                </tr>
-                <tr>
-                  <td>
-                    <ul class="list-group comments" id="responseCommentTable-1-1-1">
-                      <li class="list-group-item list-group-item-warning" id="responseCommentRow-1-1-1-1">
-                        <div id="commentBar-1-1-1-1">
-                          <span class="text-muted">
-                            From: Anonymous [Mon, 02 Feb 2015, 11:59 PM UTC]
-                          </span>
-                        </div>
-                        <div id="plainCommentText-1-1-1-1" style="margin-left: 15px;">
-                          Anonymous
-                          <span class="highlight">
-                            comment
-                          </span>
-                          to feedback Question 2
-                        </div>
-                      </li>
-                      <li class="list-group-item list-group-item-warning" id="responseCommentRow-1-1-1-2">
-                        <div id="commentBar-1-1-1-2">
-                          <span class="text-muted">
-                            From: Instructor Instructor2 Course1 [Tue, 03 Feb 2015, 11:59 PM UTC]
-                          </span>
-                        </div>
-                        <div id="plainCommentText-1-1-1-2" style="margin-left: 15px;">
-                          Instructor 2
-                          <span class="highlight">
-                            comment
-                          </span>
-                          to feedback Question 2 (Student 3 see this as anonymous
-                          <span class="highlight">
-                            comment
-                          </span>
-                          )
-                        </div>
-                      </li>
-                    </ul>
-                  </td>
-                </tr>
-              </tbody>
-            </table>
-          </div>
-        </div>
-      </div>
+  <div class="row">
+    <div class="alert alert-warning col-md-4 col-md-offset-4">
+      <img src="/images/error.png" style="float: left; margin: 0 10px 10px 0; height: 90px;">
+      <p>
+        There was an error in our server.
```

Comment:
You need to do some sanity checking on GodMode changes before committing them wholesale.